modern/src/resources/l10n/en.json

```diff
@@ -35,6 +35,7 @@
     "sharedGeofence": "Geofence",
     "sharedGeofences": "Geofences",
     "sharedCreateGeofence": "Create Geofence",
+    "sharedUploadGeofence": "Upload Geofence",
```

Comment:
By the way, I would recommend making this more generic. Maybe just "upload". It might be useful in other places in the future. And the button is on the geofences, page, so it's pretty clear already that it's for geofences.